src/index.html

```diff
@@ -1,14 +1,21 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Snake</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Snake</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Snake</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="container">
+      <div class="box box-1">1</div>
```

Comment:
```suggestion
      <div class="box">1</div>
```
Используй `:nth-child` псевдо-класс

Reply:
Привет. Ссылки поменяла, теперь се должно работать.
Немного не  поняла про "Используй :nth-child псевдо-класс".... можно чуть детальней зачем? и что сейчас не так? хочу для себя понимать механику что и какие действия улучшают/оптимизируют написание кода

Reply:
Это чтобы не нагромождать элемент дополнительными классами?